tools/update_lint_rules/lib/update_lint_rules.dart

```diff
@@ -51,14 +51,14 @@ Future<ExitStatus> updateLintRules(ProviderContainer container) async {
     await analysisOptionsService.updateDartLintRules(
       releases: dartSdkReleases,
       lintRules: lintRules.dart,
-      notRecommendedRules: notRecommendedRules,
+      notRecommendedRules: notRecommendedRules.dart,
     );
 
     final flutterSdkReleases = await sdkService.getFlutterSdkReleases();
     await analysisOptionsService.updateFlutterLintRule(
       releases: flutterSdkReleases,
       lintRules: lintRules.flutter,
-      notRecommendedRules: notRecommendedRules,
+      notRecommendedRules: notRecommendedRules.flutter,
```

Comment:
![must-badge](https://img.shields.io/badge/review-must-red.svg) 
Here, you must pass all `notRecommendedRules`, including `notRecommendedRules.dart`.

This is because Flutter's `recommended.yaml` includes Flutter's `all.yaml`, and Flutter's `all.yaml` includes Dart's `all.yaml`, which does not disable the deprecated Dart Lint rules along the way.

Reply:
@blendthink 
Thanks for pointing that out.
Fixed. (https://github.com/yumemi-inc/flutter-yumemi-lints/pull/49/commits/1b73dfdd701b971787671ad28888fca71a2c4ed9)